packages/cactus-plugin-ledger-connector-hedera/src/main/typescript/plugin-ledger-connector-hedera.ts

```diff
@@ -0,0 +1,37 @@
+console.clear();
+const {
+  AccountId,
+  PrivateKey,
+  Client,
+  AccountBalanceQuery,
+} = require("@hashgraph/sdk");
+
+// Grab the OPERATOR_ID and OPERATOR_KEY from the .env file
+const operatorId = AccountId.fromString(
+  "0.0.46018668" || process.env.OPERATOR_ID,
+);
+const operatorKey = PrivateKey.fromString(
+  "302e020100300506032b6570042204208c8e7386dd9a19356a2fb94718955f53ed30bb437e9afc1789426fa9bf90ab8fs" ||
```

Comment:
(a) don't hard code in defaults, instead it should fail
(b) don't commit private keys.  Since this is not a production key it's not a problem.  If this was a production key we would have to rotate keys.